### PR TITLE
Strengten marketing claim on download page

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

**Comment by roidelapluie** [comment]
I don't think we should make people download promql.. that just sounds sooo strange

**Comment by roidelapluie** [comment]
.. and the claim you make is probably wrong

File: content/download.html [hunk 1]
```diff
@@ -13,6 +13,11 @@ <h1>Download</h1>
       </ul>
     </div>
 
+    <p>
+      Download Prometheus, the monitoring framework and TSDB which redefined the
+      observability industry. This includes support for PromQL, the powerful query language for
+      accessing your monitoring data at a scale which was not possible before Prometheus and PromQL.
```
Comment (juliusv) [inline content/download.html:19]: How about this (just sliiightly less obnoxious?) variant:

> Download Prometheus, the leading open-source monitoring framework and TSDB. This includes support for PromQL, the powerful Prometheus query language for processing your monitoring data in a flexible way which was not possible before Prometheus and PromQL.

**Comment by roidelapluie** [comment]
Why do we need this? You also download an awesome set of service discoveries, that many other monitoring systems don't offer out of the box, and many other things.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**